src/pages/api/kakao-login.ts

```diff
@@ -1,12 +1,18 @@
 import { NextApiRequest, NextApiResponse } from "next";
 
 async function getTokenFromKakao(authCode: string) {
-  const tokenUrl = `https://kauth.kakao.com/oauth/token?grant_type=authorization_code&client_id=${process.env.NEXT_PUBLIC_REST_API_KEY}&redirect_uri=${process.env.NEXT_PUBLIC_KAKAO_REDIRECT_URI}&code=${authCode}`;
-  const response: NextApiResponse = await fetch(tokenUrl, {
-    method: "POST",
-    headers: { "Content-Type": "application/json" },
-  }).then((res) => res.json());
-  return response;
+  try {
+    const tokenUrl = `https://kauth.kakao.com/oauth/token?grant_type=authorization_code&client_id=${process.env.NEXT_PUBLIC_REST_API_KEY}&redirect_uri=${process.env.NEXT_PUBLIC_KAKAO_REDIRECT_URI}&code=${authCode}`;
+    const response: NextApiResponse = await fetch(tokenUrl, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+    }).then((res) => res.json());
+
+    console.log(response);
+    return response;
+  } catch (err) {
+    throw new Error((err as Error).message);
```

Comment:
p3
따로 에러처리를 하진 않는데 
객체를 다시 만들어서 throw는 할필요가 없어보이기도 하네용?
```suggestion
    throw err;
```